### ISSUE TITLE
Beams do not follow stem horizontal offset

### ISSUE THREAD

**Issue by bbayu123** [musescore/MuseScore#21850]
### Issue type

Engraving bug (incorrect score rendering)

### Bug description

When moving the stem offset of a note, the flags follow the stem and move to the new offset, but the beams do not. This creates a graphical anomaly that is shown in the following image.

### Steps to reproduce

1. Select a note that's part of a beamed group
2. Inside Properties: Under Note » Stem » Show more » Stem offset, alter the horizontal value of the offset

![image](https://github.com/musescore/MuseScore/assets/26726381/ce658c20-38f7-48e7-b3cb-db8214abf66c)

### Screenshots/Screen recordings

![image](https://github.com/musescore/MuseScore/assets/26726381/8012d08b-b15c-4dbd-8f5d-f2005ef1822f)

### MuseScore Version

OS: Windows 10 Version 2009 or later, Arch.: x86_64, MuseScore version (64-bit): 4.2.1-240230937, revision: github-musescore-musescore-d757433

### Regression

No

### Operating system

Windows 11

### Additional context

Potentially related to #21849, when it comes to beams not moving horizontally.

**Comment by XiaoMigros** [comment]
I think the existing behaviour is fine as is. Moving the chord means that the stem and beam follow suit, and to achieve what's pictured above the notehead offsets can be altered instead. What would no longer be possible if this 'issue'  is fixed is the ability to move stems independently of the beams, something that while not very useful seems pointless to remove

**Comment by bbayu123** [comment]
I agree that moving the noteheads would be the solution for an individual staff. However, it will affect the vertical alignment of note in the same beat when looking across the whole system. 
I would argue that vertical alignment of notes in the same beat is more important, so the stem amd beams should be moved instead of the notehead.

**Comment by XiaoMigros** [comment]
The solution I explained wouldn't cause disalignment because you'd move the chord the other direction to compensate.

**Comment by bbayu123** [comment]
I must be doing something wrong then. Please advise.
![image](https://github.com/user-attachments/assets/d4ff3140-ebc6-4ec5-a089-8f2e361d0ca1)

**Comment by Fourniture** [comment]
Another example:
![image](https://github.com/user-attachments/assets/6e1cdec9-a6d7-4008-9a85-7c7af7873e6b)

A view of the voices:
![image](https://github.com/user-attachments/assets/906d9509-d975-4418-a806-35889b53fa0a)

It seems there is no way to manually fix this example. @bkunda, if I may, I would ask for this to be fixed soon (in Musescore Studio 4.5,) as it is rather inhibiting.

**Comment by oktophonie** [comment]
![image](https://github.com/user-attachments/assets/7f22f066-3488-4ff2-b7d3-8dc3a4f42c16)
Just offset the notes, not the stems.

**Comment by Fourniture** [comment]
@oktophonie, I apologize for discussing it here, but your comment was not clear to me. I did offset the notes, but the stems stayed where they were, so I had to offset the stems as well. Am I doing something wrong?

**Comment by oktophonie** [comment]
Feel free to attach the file and we can take a look.

**Comment by Fourniture** [comment]
Here is the file (problem in measure 135): 
[Offertory.zip](https://github.com/user-attachments/files/18323572/Offertory.zip)



And here is a screen recording of the problem:



https://github.com/user-attachments/assets/4ec30d62-1649-42b6-b232-c02ab9497f60

**Comment by MarcSabatella** [comment]
Your video clearly shows you offsetting the notehead only - that’s the label on the field you are changing.  To offset the full note, use the control under Appearance.  Or - much simpler - enter Edit mode (press F2) and use the cursor keys.

**Comment by Fourniture** [comment]
Thank you. I wish the process was more evident...

**Comment by MarcSabatella** [comment]
Indeed.  But anyhow, I definitely encourage you to ask for help on the support forums over on MuseScore.org when you have questions about how to do something or find something isn’t working as expected.

**Comment by davidstephengrant** [comment]
Closing as by design.
